tool/site/client/src/components/messageExpanders.tsx

```diff
@@ -45,6 +48,7 @@ const MessageExpanders: React.FC<MessageExpandersProps> = ({
 interface MessageExpanderProps {
   name: string
   message: string
```

Comment:
kinda of annoying we are passing in basically the same thing twice, just one typed. can we make do without the prop `message`?

Reply:
the `message` prop has to exist as we need to pass the `xml` data as a string, I could get rid of the `jsonMessage` potentially and have something like:

```
let displayMessage: FhirResponse | string

try {
  displayMessage = JSON.parse(message)
} catch {
  displayMessage = message
}

// render message
{ typeOf(message) === string ? <xml string> : <json view component> }
```

Reply:
changed this to be two distinct components